common/djangoapps/student/tests/test_tasks.py

```diff
@@ -91,3 +92,27 @@ def test_UnrecoverableSendError(self, mock_log):
         self.assertEqual(mock_log.info.call_count, 0)
         self.assertEqual(mock_log.error.call_count, 0)
         self.assertEqual(mock_log.exception.call_count, 1)
+
+    @mock.patch('student.tasks.log')
+    @mock.patch('student.tasks.ace.send', mock.Mock(side_effect=ChannelError))
+    @mock.patch('common.djangoapps.student.views.management.theming_helpers.get_current_site')
+    @mock.patch('openedx.core.djangoapps.site_configuration.helpers.get_current_site_configuration')
+    def test_from_address_in_send_email(self, mock_site_configuration, mock_get_current_site, mock_log):
+        """
+        Tests that a from address pull from site configuration.
```

Comment:
`Tests that the "from_address" is pulled from the site configuration.`

Reply:
Done